lib/internal/Magento/Framework/Amqp/Topology/ArgumentProcessor.php

```diff
@@ -17,22 +22,25 @@ trait ArgumentProcessor
      * @param array $arguments
      * @return array
      */
-    public function processArguments($arguments)
+    public function processArguments($arguments): array
     {
         $output = [];
         foreach ($arguments as $key => $value) {
             if (is_array($value)) {
                 $output[$key] = ['A', $value];
             } elseif (is_int($value)) {
```

Comment:
In this case, using only `is_numeric` will be enough.
Thanks

Reply:
you mean in scope of this PR is_int check should be removed?

Reply:
up

Reply:
@bartoszkubicki Yes, you are right, is_int check can be removed.